src/coreclr/tools/aot/ILCompiler.RyuJit/ILCompiler.RyuJit.csproj

```
@@ -68,8 +68,8 @@
     <Compile Include="..\..\Common\JitInterface\TypeString.cs">
       <Link>JitInterface\TypeString.cs</Link>
     </Compile>
-    <Compile Include="..\..\Common\JitInterface\CorInfoBase.cs">
-      <Link>JitInterface\CorInfoBase.cs</Link>
+    <Compile Include="..\..\Common\JitInterface\CorInfoBase_generated.cs">
```

```suggestion
    <Compile Include="..\..\Common\JitInterface\CorInfoImpl_generated.cs">
```
The type that lives in the generated file is called `CorInfoImpl` so this is more appropriate name.